ZConfig/sphinx.py

```diff
@@ -149,16 +149,18 @@ def printSchema(self):
 
     class SchemaToRstDirective(Directive):
         required_arguments = 1
-        optional_arguments = 2
+        optional_arguments = 4
```

Comment:
What does this do?

Reply:
Not what I thought it did! Turns out I was wrong when I first wrote this code, I thought it had something to do with the options, but it doesn't. *arguments* are the thing on the line with the directive, *options* are the things that come after it. So `.. zconfig:: ZConfig foo bar` has three arguments. This really needs to be 0 because we don't take any argument other than a package.